protocol-tokio/src/lib.rs

```diff
@@ -1,14 +1,15 @@
 #[macro_use]
 extern crate cbor_event;
 extern crate bytes;
-extern crate cardano;
 extern crate tokio_codec;
 extern crate tokio_io;
 #[macro_use]
 extern crate futures;
 #[macro_use]
 extern crate log;
 
+extern crate network_core;
```

Comment:
I hope this is temporary and we will eventually de-spaghettify the crates by merging protocol-tokio with network-ntt.